airbyte-cdk/java/airbyte-cdk/config-models-oss/build.gradle

```diff
@@ -31,30 +31,3 @@ jsonSchema2Pojo {
 tasks.register('generate').configure {
     dependsOn tasks.named('generateJsonSchema2Pojo')
 }
-
-test {
-    useJUnitPlatform {
-        excludeTags 'log4j2-config', 'logger-client'
-    }
-    testLogging {
-        events "passed", "skipped", "failed"
-    }
-}
-
-tasks.register('log4j2IntegrationTest', Test) {
-    useJUnitPlatform {
-        includeTags 'log4j2-config'
-    }
-    testLogging {
-        events "passed", "skipped", "failed"
-    }
-}
-
-tasks.register('logClientsIntegrationTest', Test) {
-    useJUnitPlatform {
-        includeTags 'logger-client'
-    }
-    testLogging {
-        events "passed", "skipped", "failed"
-    }
-}
```

Comment:
These tasks were not doing anything. We don't have any of these junit tags defined anywhere. Presumably, this is a legacy from before the platform was spun out of this repo.